pyanaconda/payload/dnfpayload.py

```diff
@@ -412,80 +414,149 @@ def addRepo(self, ksrepo):
         super(DNFPayload, self).addRepo(ksrepo)
 
     def _apply_selections(self):
```

Comment:
This method could be probably split into several simpler ones, that can be tested with unittests.

Reply:
I'm not sure this is required anymore with the common include/exclude sets as there are now just short pieces of code filling one of the sets.